digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/ProjectDAO.scala

```diff
@@ -246,7 +246,7 @@ object ProjectDAO {
     }
     val query =
       s"""SELECT id, state, name, created_by, created_date, start_date, modified_by, modified_date, add_info, status_info, ely
-          FROM project $filter order by name, id """
+          FROM project $filter where created_date is not null and modified_date is not null and start_date is not null order by name, id """
```

Comment:
This would filter out all unmodified projects from project listing. It'd be better to
a) return datetimes as option[datetime]s
b) change table definition and require created_date to be not null -> can't be null and return modified_date as COALESCE(modified_date, created_date)